colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala

```diff
@@ -36,13 +35,11 @@ class MetricInterval private[metrics](val namespace : MetricAddress, val interva
  */
 case class MetricSystem private [metrics](namespace: MetricAddress, metricIntervals : Map[FiniteDuration, MetricInterval]) {
 
-  def sharedCollection(globalTags: TagMap = TagMap.Empty)(implicit fact: ActorRefFactory) = {
-    implicit val me = this
-    SharedCollection(globalTags)
-  }
+  implicit val base = new Collection(CollectorConfig(metricIntervals.keys.toSeq))
```

Comment:
I just realized that on the apply for the MetricSystem object, metricIntervals is a Seq not a Set, we should probably fix that.